port/cpl_mask.h

```diff
@@ -0,0 +1,136 @@
+/**********************************************************************
+ * $Id$
```

Comment:
Does GDAL still need to carry the CVSish `$Id`?

Reply:
indeed. mass removal in https://github.com/OSGeo/gdal/pull/6540